docs/cep/reference/extensions/execution/fx.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Function Worker(Fx)
+---
+
+The `fx` extension executes an Edge Worker.
+
+## Syntax
+
+Executes parameterless function:
+
+    <OBJECT> fx(<STRING> name)
+
+Executes function with parameters set via JSON or object:
+
+    <OBJECT> fx(<STRING> name, <STRING|OBJECT> json)
+
+Executes function with parameters set as pairs of arguments:
+
+    <OBJECT> fx(<STRING> name, <STRING> param1Name, <String|Bool|Int|Long|Double|Float> param1Value, ...)
+
+Examples of function with different syntax:
+```js
+	fx('fn-name')
+    fx('fn-name', \"{name : 'John', age : 23}\")
+    fx('fn-name', json:getObject(\"{name : [{name:'John'}], age : 23}\",'$'))
+    fx('fn-name', 'name', 'John', 'age', 23)
+```
+
+## Example
+
```

Comment:
This example needs an introduction to provide context.

Reply:
Added an introduction